codice/alliance
# CAL-399 Add support for the EXPLTB TRE

#### What does this PR do?
Add support for the EXPLTB TRE, so that when ingesting a NITF containing this TRE, the metacard will be populated with its data.
#### Who is reviewing it? 
@harrison-tarr 
@derekwilhelm 
@dcruver  
@ryeats 
(please choose AT LEAST two reviewers that need to approve the PR before it can get merged; if a component team is listed, at least one of its members needs to approve)
#### Choose 2 committers to review/merge the PR.
(please choose ONLY two committers from below, delete the rest)
@bdeining
@jaymcnallie 
#### How should this be tested?
Run a full build. Start up Alliance and ingest a NITF with the EXPLTB TRE and verify the metacard contains "ext.nitf.expltb.*" fields.
#### What are the relevant tickets?
[CAL-399](https://codice.atlassian.net/browse/CAL-399)
#### Screenshots (if appropriate)
#### Checklist:
